test/jdk/java/lang/ProcessHandle/InfoTest.java

```diff
@@ -307,10 +314,15 @@ public static void test3() {
                     Assert.assertTrue(exe.exists(), "command must exist: " + exe);
                     Assert.assertTrue(exe.canExecute(), "command must be executable: " + exe);
                 }
+
                 if (info.arguments().isPresent()) {
                     String[] args = info.arguments().get();
                     if (args.length > 0) {
-                        Assert.assertEquals(args[0], String.valueOf(sleepTime));
+                        if (timeIsLastParam) {
```

Comment:
I gotta ask: is it true that time argument is the last param in _all_ testing modes? If so, we don't need `timeIsLastParam` at all?

Reply:
Hah, yes. I was trying too hard to not change the default behavior.